pq-crypto/README.md

```diff
@@ -48,4 +48,21 @@ and use the BIKE1_L1.const.kat from the above Additional_Implementation.2019.03.
 1. Add fuzz testing in `tests/fuzz/s2n_KEM_NAME_fuzz_test.c`
 1. Add formal verification in `tests/saw/KEM_NAME/verify.saw`
 1. Create a new `s2n_cipher_suite` in `tls/s2n_cipher_suites.c`
-1. Create a new `s2n_cipher_preferences` in `tls/s2n_cipher_prefrences.c` that uses the new cipher suite
\ No newline at end of file
+1. Create a new `s2n_cipher_preferences` in `tls/s2n_cipher_prefrences.c` that uses the new cipher suite
+
+## How to use PQ cipher suites
+1. Checkout s2n `git clone https://github.com/awslabs/s2n.git`
+1. Following the docs/USAGE-GUIDE.md build s2n
+1. Use the sample server and client in the bin directory:
+```bash
+# Terminal 1
+export LD_LIBRARY_PATH=/path-to-s2n/s2n/test-deps/openssl-1.1.1/lib:/path-to-s2n/s2n/test-deps/openssl-1.1.1/lib:/path-to-s2n/s2n/lib:/path-to-s2n/s2n/bin
+export PATH=/path-to-s2n/s2n/bin:$PATH
+s2nd --cert tests/pems/rsa_2048_sha256_wildcard_cert.pem --key tests/pems/rsa_2048_sha256_wildcard_key.pem --negotiate --ciphers KMS-PQ-TLS-1-0-2019-06 0.0.0.0 8888
```

Comment:
I'd add some comments here to explain what `s2nd` and `s2nc` are.
`#Use s2nd CLI tool to start a TLS daemon with KMS-PQ-TLS-1-0-2019-06 Cipher Preferences`